Source/control.cpp

```diff
@@ -684,6 +686,7 @@ void DrawSpellList(const Surface &out)
 	auto &myPlayer = Players[MyPlayerId];
 
 	for (auto &spellListItem : GetSpellListItems()) {
+		const spell_id spell = spellListItem.id;

```

Comment:
Can we rename the variable to `spellId`?

Reply:
Done